internal/v1/api.yaml

```diff
@@ -1125,6 +1132,15 @@ components:
       type: string
       enum: ["api", "ui"]
       default: "api"
+    Targets:
+      required:
+        - ImageTypes
+      type: object
+      properties:
+        ImageTypes:
+          type: array
+          items:
+            type: string
```

Comment:
The `ImageTypes` element appears to be required, should not be required = build all targets.

Please make use of the existing `components/schemas/ImageTypes` type which is in fact enum. Note enums are pain to work with in Go since the language does not support enums and I personally would rather avoid enums in OpenAPI, but since the project already does use them, let’s stick with it.

Please write documentation for this endpoint now, add at least two sentences explaining the behavior.